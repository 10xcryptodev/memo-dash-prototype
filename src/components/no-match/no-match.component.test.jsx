import React from 'react'
import { shallow } from 'enzyme'
import NoMatch from './no-match.component'

describe('<NoMatch />', () => {
  let wrapper

  beforeEach(() => {
    const div = document.createElement('div')
    document.body.appendChild(div)
    wrapper = shallow(<NoMatch location={{ pathname: '/test' }} />)
  })

  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
