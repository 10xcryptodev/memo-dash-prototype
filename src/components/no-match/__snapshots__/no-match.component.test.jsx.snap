// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> renders without crashing 1`] = `
<div>
  <h3>
    No match for 
    <code>
      /test
    </code>
  </h3>
</div>
`;
